src/runner/UpdateUtils.cpp

```diff
@@ -218,6 +218,22 @@ void CheckForUpdatesCallback()
         UpdateState::store([&](UpdateState& v) {
             v = std::move(state);
         });
+
+        if (state.state == UpdateState::State::upToDate)
```

Comment:
What happened if inet connection is off. PT settings say "up to date" in this case. Can we get problems when update isn't done at this time?

Reply:
Nice catch!